src/style.css

```diff
@@ -1 +1,28 @@
-/* styles go here */
+.field {
+  margin: 10px 0;
+}
+
+.fieldset {
+  margin: 20px 0;
+}
+
+.fieldsetTop {
+  margin-top: 0;
+}
+
+.fieldset > :nth-child(2) {
+  margin-top: 0;
+}
+
+.input {
+  margin: 0;
+  padding: 0;
+}
+
+.selectRecommend {
+  margin-top: 10px;
+}
+
+.fieldset > :last-child {
+  margin-bottom: 0;
+}
```

Comment:
There are too many styles. You can just add class `label` to each label and style it with 
`.label {`
  `display: block;`
  `margin-bottom: 10px;`
`}`

`.label:last-child {`
  `margin-bottom: 0;`
`}`

Reply:
but in checklist written: "Remember, any other styles besides margins, should be browser default.", or "display: block;"  exception?